db/sqlc/db_manager.go

```diff
@@ -0,0 +1,17 @@
+package sqlc
+
+import "time"
+
+const (
+	QuerierCtxTimeout = time.Second * 10
+)
+
+type DbManager struct {
```

Comment:
`DbManager` manages the db operations, and act as a middle man between server and db. So having the analytics manager within this struct does not make sense. Usually, Analytics managers/engines are their standalone objects in servers (like `gameManager` or `sessionManager`). We could initialize our Analytics manager on `main.go` and pass the db queries into it. You can see why this approach won't work when you start adding different sql tables, and you would observer the db manager gets bigger and biiiigger and biiiiiiiger 😄

Reply:
lol I see your point.
I'm actually thinking that this would be a problem other way around too. Imagine we have a lot of queries. Then we're gonna have a lot of stand alone objects similar to `AnalyticsManager`.

`Queries` is already publicly exposed by sqlc and we can make the functions that execute the querier as clear as we want. 
I personally think this level of middle man is unnecessary

Reply:
We can just use `Queries` directly since it's clear enough what we want.
e.g.: queries.IncrementGamesCreatedCount()

This clearly states what we're doing.

Reply:
Yeah I agree we don't need the middle man. We can just use `queries`